cypress/e2e/dashboard.cy.ts

```diff
@@ -65,4 +73,45 @@ describe('Dashboard', () => {
     cy.contains('42.00').should('not.exist')
     cy.contains('12.00')
   })
+
+  it('links to the envelope filtered transactions', function () {
+    cy.wrap(
+      Cypress.Promise.all([
+        createAccount({ name: 'Internal' }, this.budget),
+        createAccount({ name: 'External', external: true }, this.budget),
+      ]).then(([internalAccount, externalAccount]) => {
+        createTransaction(
+          {
+            sourceAccountId: internalAccount.id,
+            destinationAccountId: externalAccount.id,
+            envelopeId: this.firstEnvelope.id,
+            note: 'First Transaction',
+            amount: 1,
+          },
+          this.budget
+        )
+        createTransaction(
+          {
+            sourceAccountId: internalAccount.id,
+            destinationAccountId: externalAccount.id,
+            envelopeId: this.secondEnvelope.id,
+            note: 'Second Transaction',
+            amount: 2,
+          },
+          this.budget
+        )
+      })
+    )
+
+    // We need to reload to load everything from the backend, not just the transactions
```

Comment:
What do you mean by „not just the transactions“? If we just stay on /, the transactions shouldn’t be loaded either.

Reply:
Transactions are loaded when switching to the transaction view, but the accounts are not since they are passed from the props.